src/applications/mhv/medications/containers/Prescriptions.jsx

```diff
@@ -146,11 +153,20 @@ const Prescriptions = () => {
 
   useEffect(
     () => {
-      if (prescriptions) {
-        setPrescriptionsPdfList(buildPrescriptionsPDFList(prescriptions));
+      dispatch(getSortedListNoPagination(sortEndpoint || defaultSortEndpoint));
+    },
+    [sortEndpoint, dispatch, defaultSortEndpoint],
+  );
```

Comment:
This needs to be done only on Download PDF button click I believe...

Reply:
yes, that makes better sense actually. Thanks for noting that!